support-frontend/assets/components/marketingConsent/marketingConsentWithCheckbox.jsx

```diff
@@ -112,7 +113,25 @@ class MarketingConsentWithCheckbox extends Component<PropTypes, StateTypes> {
   }
 
   render() {
-    const emailFrequency = this.props.countryGroupId === 'AUDCountries' ? 'an occasional' : 'a weekly';
+    const createTitleText = () => {
+      const { countryGroupId, contributionType } = this.props;
+      const titleTexts = {
```

Comment:
I think this can be declared outside of the class - somewhere above it

Reply:
👍